client/app/components/navbar/navbar-notification.service.js

```diff
@@ -0,0 +1,109 @@
+class NavbarNotificationService {
+    constructor ($interval, $q, $translate, Alerter, constants, OvhApiNotificationAapi, translator, UNIVERSE) {
+        this.$interval = $interval;
+        this.$q = $q;
+        this.$translate = $translate;
+        this.alerter = Alerter;
+        this.constants = constants;
+        this.OvhApiNotificationAapi = OvhApiNotificationAapi;
+        this.translator = translator;
+        this.UNIVERSE = UNIVERSE;
+
+        this.NOTIFICATION_REFRESH_TIME = 60000;
+    }
+
+    getMessages () {
+        return this.$translate.refresh().then(() => this.OvhApiNotificationAapi.query({
+            lang: this.$translate.preferredLanguage(),
+            target: this.constants.target,
+            universe: this.UNIVERSE
+        }).$promise);
+    }
+
+    getSubLinks () {
+        return this.getMessages().then((messages) => messages.map((message) => this.convertSubLink(message))).catch(() => [{
+            id: "0",
+            date: moment(),
+            time: this._formatTime(moment()),
+            level: "error",
+            subject: this.translator.tr("common_navbar_notification_error_subject"),
+            description: this.translator.tr("common_navbar_notification_error")
+        }]);
+    }
+
+    /* eslint-disable class-methods-use-this */
```

Comment:
Maybe create a static method instead ?

Reply:
because I cannot start a static method with underscore (eslint rule). I javascript I found that static functions are best when called outside of the main class like `myObject.staticFunction()`, but inside you would have to call `MyObject.staticFunction()`